backend/src/StamAcasa.EmailService/EmailBuilder/Models/EmailRequestModel.cs

```diff
@@ -7,5 +8,6 @@ public class EmailRequestModel
         public string Address { get; set; }
         public string Type { get; set; }
         public Dictionary<string, string> PlaceholderContent { get; set; }
+        public EmailTemplate TemplateType { get; set; }
     }
```

Comment:
`IdentityServer` and `NotificationService` will send requests to `EmailService` via MQ.
Would it make sense to move these definitions in `Common`?